GSA/digitalgov.gov
# Create contact center community landing page

## Summary

Creates a community landing page for the contact center community. 

IMPORTANT: 
This should not merge until the logo is updated and we have a go-ahead for the redirect. 

### Preview

[Link to Contact center community preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/bc-contact-center-community-landing/communities/contact-center/)

<img width="877" alt="Screenshot 2024-06-27 at 3 41 35 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/a08027c1-3b0b-40b7-a189-5686db637843">


### Solution

### How To Test

1. Review for typos or formatting errors 
2. Test links
3. Make sure that the logo appears on the [community landing page](https://digital.gov/communities/)
4. Make sure that the hover color #01538f  appears on the [community landing page](https://digital.gov/communities/) "Join the Contact Center Community" button.


---

### Dev Checklist
